app/setup_handlers.go

```diff
@@ -5,12 +5,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	authoritytypes "github.com/zeta-chain/zetacore/x/authority/types"
-	lightclienttypes "github.com/zeta-chain/zetacore/x/lightclient/types"
-	observertypes "github.com/zeta-chain/zetacore/x/observer/types"
 )
 
-const releaseVersion = "v17"
```

Comment:
Do we really need to do a full major version for this change? This seems more like a v17.1.0 but requires an upgrade proposal.

Reply:
This is a consensus breaking changes and we decided to do a major release for every consensus breaking changes, unless we discuss changing this recently?

Reply:
Ok